specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -238,13 +240,17 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
           </div>
         )}
         isDependent={isDependent}
+        // TODO: Figure out what to do with this line. 
+        // ResourceView in this file was never passed a isInRecordSet prop until this PR
+        // but isNewRecordSet has been removed in https://github.com/specify/specify7/pull/4904
+        // Can possibly just pass on the isInRecordSet prop received by this component?
         isInRecordSet={!isNewRecordSet}
         isLoading={isLoading || isExternalLoading}
         isSubForm={false}
         resource={resource}
         title={title}
         viewName={viewName}
-        onAdd={handleClone}
+        onAdd={(resources) => resources.map(r => handleClone?.(r))}
```

Comment:
make handleClose accept an array here too

```suggestion
        onAdd={handleClone}
```